packages/expo-media-library/android/src/main/java/expo/modules/medialibrary/albums/AlbumUtils.kt

```diff
@@ -51,16 +47,12 @@ fun queryAlbum(
         putString("title", albumsCursor.getString(bucketDisplayNameIndex))
         putInt("assetCount", albumsCursor.count)
       }
-      promise.resolve(result)
+      return result
     }
   } catch (e: SecurityException) {
-    promise.reject(
-      ERROR_UNABLE_TO_LOAD_PERMISSION,
-      "Could not get albums: need READ_EXTERNAL_STORAGE permission.",
-      e
-    )
+    throw UnableToLoadException("Could not get albums: need READ_EXTERNAL_STORAGE permission $e")
   } catch (e: IllegalArgumentException) {
-    promise.reject(ERROR_UNABLE_TO_LOAD, "Could not get album.", e)
+    throw UnableToLoadException("Could not get album $e")
```

Comment:
```suggestion
    throw UnableToLoadException("Could not get album: ${e.message}", e)
```
You can pass `cause` to the coded exception to make it more readable for the future.